tests/test_qnode.py

```diff
@@ -596,6 +596,7 @@ def qf(x, y):
             qml.Squeezing(0.3, x, wires=[1])  # x affects gates on both wires, y unused
             qml.Rotation(1.3, wires=[1])
             return qml.expval.X(0), qml.expval.X(1)
+
         check_methods(qf, {0:'F'})
 
         def qf(x, y):
```

Comment:
We could define these outside the class as qf1, qf2 etc., just a suggestion. I understand that we cannot avoid making QNodes and circuits as they are function based but maybe for such indpendent q functions we can just put them in the beginning of the tests?

Reply:
I'm on the fence - personally, I prefer having the quantum functions defined in the test function - this makes debugging clearer, and each test more self contained. An alternative is that if the same qnode is re-used over and over in tests, it could be moved to a test fixture

Reply:
I guess then for the tests it is okay. Since this is not part of the main code. But test fixtures seems like a neater way to do it. We can go ahead with this.